imap_data_access/file_validation.py

```diff
@@ -205,7 +255,7 @@ def extract_filename_components(filename: str | Path) -> dict:
         pattern = (
             r"^(?P<mission>imap)_"
             r"(?P<instrument>[^_]+)_"
-            r"(?P<datalevel>[^_]+)_"
```

Comment:
can you also fix `startdate` to `start_date` and `enddate` to `end_date`?

Reply:
Sorry, missed that in your first comment!